documentation/assemblies/assembly-upgrading.adoc

```diff
@@ -1,20 +1,23 @@
 // This assembly is included in the following assemblies:
 //
 // assembly-service-admin-guide.adoc
+
 [id='upgrading-{context}']
 = Upgrading {ProductName}
 
 {ProductName} supports upgrades between minor versions using cloud native tools and the same
-mechanism used to apply configuration changes.  When upgrading, the updated infrastructure
-configuration of the new version will trigger the upgrade to start.
+mechanism used to apply configuration changes. When upgrading, the updated infrastructure
+configuration of the new version triggers the upgrade process to begin.
 
```

Comment:
First sentence is not too clear.  Is the first clause worthwhile?  It is the same tooling we introduced the reader to in the install guide. With the second clause we are trying to recommend that the user selects the same upgrade mechanism (ansible or bundle) as they did for the install, but I don't think this comes across clearly.

Second sentence is no longer correct.  It should say "applying the configuration change automatically triggers the upgrade process to start".  It is no longer trigger by an update to the infraconfig.

Reply:
@k-wall, Does the release template method still work, or does that need to be removed?

Reply:
Are you referring to the bundles? If so, yes, they still work/supported.